tests/storage/restore.go

```diff
@@ -740,6 +740,7 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 					snapshotStorageClass,
 				))
 
+				Expect(console.BlockDeviceResizeExpecter(vmi)).To(Succeed())
```

Comment:
The importance of seeing the resize command from the kernel is not clear to me. Could you elaborate this a little?

Reply:
@ShellyKa13 just to understand this: we see a full login prompt (cirros), and then we try to mount the disks, but until we see this message we would fail on that? Is that what this is about?

Reply:
@rmohr before the snapshot we write to some file to check for it later. In this test for some reason when doing the batch of commands which we expect empty prompy as a response of some of them we instead get the resize message which fails the test. this resize message is supposedly part of the start of the vm and for some reason we get the successe response late (happened only in the ci). I tried to make sure that we got the resize message before continuing to the doRestore